frontend/lib/resolution/scope-types.cpp

```diff
@@ -85,6 +85,26 @@ int BorrowedIdsWithName::countVisibleIds() {
   return count;
 }
 
+bool BorrowedIdsWithName::containsOnlyMethodsOrFields() const {
+  if (moreIdvs_ == nullptr) {
+    if (isIdVisible(idv_)) {
+      return idv_.isMethodOrField();
+    }
+  }
+
+  bool onlyMethodsFields = true;
+  for (const auto& idv : *moreIdvs_) {
+    if (isIdVisible(idv)) {
+      if (!idv.isMethodOrField()) {
+        onlyMethodsFields = false;
+        break;
+      }
+    }
+  }
+
+  return onlyMethodsFields;
```

Comment:
why not `return true` / `return false` instead of the `onlyMethodFields` temporary?